playground/mocks/spec-okta-api/api/v1/registration/form.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable @okta/okta/no-unlocalized-text */
+const templateHelper = require('../../../../config/templateHelper');
+
+module.exports = [
+  templateHelper({
+    path: '/api/v1/registration/form',
+    method: 'GET',
+    template: JSON.stringify({
```

Comment:
Hi @lesterchoi-okta , I assume specify `template` isn't required (or actually be abundant) ?

Reply:
@haishengwu-okta yes, think you're right. i'm going to add a cleanup task and follow up right after release